aten/src/ATen/native/cuda/IndexKernel.cu

```diff
@@ -93,6 +97,73 @@ static void index_put_kernel(TensorIterator& iter, IntArrayRef index_size, IntAr
   });
 }
 
+template<typename scalar_t, typename mask_t>
+__global__ void masked_select_out_cuda_kernel(
```

Comment:
You don't need the kernel anymore, do you?

Reply:
Nope, nor the `#include`s I added. Removed them